.github/workflows/R-CMD-check.yaml

```diff
@@ -0,0 +1,52 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [main, master, v3_dev]
+  pull_request:
+    branches: [main, master, v3_dev]
```

Comment:
Can't we delete the v3_dev and main branch from here??

Reply:
also can we add  ` workflow_dispatch:` so we can manually launch the test when needed it comes in handy sometimes and can we add back the block of code which triggers a scheduled run? that way if the main branch is left untouched for awhile it will still be run at least once a month as a precaution to check to see if changes in dependencies break hector?
```
  schedule:
   # * is a special character in YAML so you have to quote this string, it is set up to run at 6am
   # on the first day of the month, this is helpful for periods of time when PRs are not being merged.
    - cron:  '0 6 1 * *'
```